fe1-web/components/DeleteButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {
+  StyleSheet, Image, Pressable, ImageStyle,
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+const trashcan = require('res/img/delete.svg');
+
+/**
+ * Delete button that displays a little trashcan and executes an onPress action given in props
+ */
+
+const styles = StyleSheet.create({
+  icon: {
+    width: 26,
+    height: 26,
+  } as ImageStyle,
+});
+
+function DeleteButton({ action }: IPropTypes) {
+  return (
+    <Pressable onPress={action}>
+      <Image style={styles.icon} source={trashcan} />
+    </Pressable>
+  );
+}
```

Comment:
Good! One little thing maybe: have you tried modifying the user cursor whenever you hover over the trashcan? Whenever a link/button/... is clickable, the cursor generally changes to the "hand cursor". Can we do this here too?

I didn't look into how to do it using react, but I'm asking this because I know that it is veeery easy to do it in CSS (closely related to the way react works)

Reply:
Really good point. Added it by doing

```const styles = StyleSheet.create({
  icon: {
    width: 26,
    height: 26,
    cursor: 'pointer',
  } as ImageStyle,
});